lightkurve/search.py

```diff
@@ -0,0 +1,530 @@
+from __future__ import division, print_function
+import os
+import sys
+import logging
+import numpy as np
+import copy
+
+from astroquery.mast import Observations
+from astroquery.exceptions import ResolverError
+from astropy.table import Table, unique
+from astropy.coordinates import SkyCoord
+from astropy.io import ascii
+from .lightcurve import KeplerLightCurve, TessLightCurve
```

Comment:
We can remove this line because `search.py` never returns or uses `LightCurve` objects